src/components/Spawn/editHostModal/getFormSchema.tsx

```diff
@@ -229,14 +228,12 @@ export const getFormSchema = ({
       },
     },
     instanceType: {
-      "ui:widget": AntdSelect,
```

Comment:
It might be good to add a state when there are no options available:

<img width="418" alt="Screen Shot 2022-12-12 at 12 42 42 PM" src="https://user-images.githubusercontent.com/47064971/207116024-b63a742d-fa87-4ee4-918c-22134d44ff80.png">

Reply:
Done 
<img width="500" alt="image" src="https://user-images.githubusercontent.com/4605522/207131376-79a2049c-1ccf-42a1-8288-261e0a7ce06b.png">